### ISSUE TITLE
Request for code or details on metric calculation

### ISSUE THREAD

**Issue by SYuan-X** [Harxis/G2Face#5]
Hi, thank you for sharing your great work!

I was wondering if you could provide the code (or more implementation details) for calculating the metrics reported in the following tables in your paper. For example, I noticed that there are multiple models trained on MS1MV3 with different backbones. Could you also share which the pretrained models (and their backbones) and  image preprocessing steps you used in your work? 

Thank you very much for your time and help!

<img width="492" height="194" alt="Image" src="https://github.com/user-attachments/assets/0bd0d0df-f449-402f-8fbe-67bfa5030b54" />

<img width="1000" height="480" alt="Image" src="https://github.com/user-attachments/assets/34541d77-0816-4e3d-998e-70c6edb6d8c5" />

**Comment by Harxis** [comment]
Thank you for your attention. We generally implement our metrics according to the official repositories of various measurement methods. For different face recognition models, we use the weights of the IRESNet 101 backbone based on the dataset reported in the paper.
